alerter.c

```diff
@@ -3,29 +3,72 @@
 
 int alertFailureCount = 0;
 
+typedef enum
+{
+   Production,
+   Test
+}Environment;
+
 int networkAlertStub(float celcius) {

```

Comment:
stub code is part of same file, which means that this code too will get shipped to field. Is it intended?